test/integration/wdio.conf.js

```diff
@@ -4,8 +4,6 @@
 const testConfig = require('../testConfig');
 const fs = require('fs');
 
-const CHROME_BINARY = require('puppeteer').executablePath();
```

Comment:
wdio fetches whatever version you tell it to, now, and does the pathing itself.

This might also make the puppeteer dependency obsolete, will check in a later PR.